src/CSharp/MetadataWebApi/MetadataWebApi.Tests/IMetadataApiExtensionsTests.cs

```diff
@@ -25,29 +24,5 @@ public static class IMetadataApiExtensionsTests
             // Act and Assert
             await Assert.ThrowsAsync<ArgumentNullException>("value", () => value.GetDownloadUriAsync(fileName, fileHash));
         }
-
```

Comment:
@Lexcess Didn't consider deleting the other test while you were at it? Not sure how much value checks for ArgumentNullExceptions bring.